docs/configuration.md

```diff
@@ -644,6 +645,7 @@ var config = {
 >  muted: false,
 >  pictureInPicture: true,
 >  playAdsWithMSE: false,
+>  forceBreakStall: false,
```

Comment:
lets move to to `streaming` section

Reply:
There's no `streaming` section, need to create one